setup.sh

```diff
@@ -100,58 +111,94 @@ check() {
         exit 1
     fi
 
+    # setup environment file
     if [ ! -f "_env" ]; then
-        echo "Creating a configuration file..."
-        cp _env.example _env
+        echo '# Environment variables for for WordPress site' > _env
+        echo '# please include the scheme http:// or https:// in the URL variable' >> _env
+
+        echo 'WORDPRESS_URL=http://'wordpress.svc.${TRITON_ACCOUNT}.${TRITON_DC}.cns.triton.zone >> _env
```

Comment:
I think we want to set this to the nginx service name, with it set to wordpress.svc WordPress is going to generate links using that URL and will be sending any internal clicks directly to the wordpress containers rather than through the nginx LB